src/main/java/org/gwtproject/tools/cldr/DateTimeFormatInfoProcessor.java

```diff
@@ -35,627 +27,613 @@
-    localeData.addEntries("dayPeriod-abbrev", cldrFactory,
-        "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dayPeriods/"
-            + "dayPeriodContext[@type=\"format\"]/"
-            + "dayPeriodWidth[@type=\"abbreviated\"]/dayPeriod[@type=\"am\"]", "dayPeriod", "type");
-    localeData.addEntries("dayPeriod-abbrev", cldrFactory,
-        "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dayPeriods/"
-            + "dayPeriodContext[@type=\"format\"]/"
-            + "dayPeriodWidth[@type=\"abbreviated\"]/dayPeriod[@type=\"pm\"]", "dayPeriod", "type");
-
-    localeData.addEntries("era-abbrev", cldrFactory,
-        "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/eras/eraAbbr", "era", "type");
-    localeData.addEntries("era-wide", cldrFactory,
-        "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/eras/eraNames", "era", "type");
-    localeData.addDateTimeFormatEntries("date", cldrFactory);
-    localeData.addDateTimeFormatEntries("time", cldrFactory);
-    localeData.addDateTimeFormatEntries("dateTime", cldrFactory);
-    loadWeekData();
-    loadFormatPatterns();
-  }
-
-  protected void writeOneJavaFile(String path, String packageName, String className,
-      GwtLocale locale) throws IOException {
-    // TODO(jat): make uz_UZ inherit from uz_Cyrl rather than uz, for example
-    System.out.println("Generating DateTimeFormatInfo for locale : "+locale.getAsString());
-
-    GwtLocale parent = localeData.inheritsFrom(locale);
-    PrintWriter pw = createOutputFile(path);
-    printHeader(pw);
-    pw.print("package " + packageName + ";");
-    // GWT now requires JDK 1.6, so we always generate @Overrides
-    setOverrides(true);
-    pw.println();
-    pw.println("// DO NOT EDIT - GENERATED FROM CLDR AND ICU DATA");
-    pw.println();
-    if (locale.isDefault()) {
-      pw.println("/**");
-      pw.println(" * Default implementation of DateTimeFormatInfo interface, "
-          + "using values from");
-      pw.println(" * the CLDR root locale.");
-      pw.println(" * <p>");
-      pw.println(" * Users who need to create their own DateTimeFormatInfo "
-          + "implementation are");
-      pw.println(" * encouraged to extend this class so their implementation "
-          + "won't break when   ");
-      pw.println(" * new methods are added.");
-      pw.println(" */");
-    } else {
-      pw.println("/**");
-      pw.println(" * Implementation of DateTimeFormatInfo for the \"" + locale + "\" locale.");
-      pw.println(" */");
+
+    protected void writeOneJavaFile(String path, String packageName, String className,
+                                    GwtLocale locale) throws IOException {
+        // TODO(jat): make uz_UZ inherit from uz_Cyrl rather than uz, for example
+        System.out.println("Generating DateTimeFormatInfo for locale : " + locale.getAsString());
+
+        GwtLocale parent = localeData.inheritsFrom(locale);
+        PrintWriter pw = createOutputFile(path);
```

Comment:
would be lovely to retrofit to use javapoet, but i think you made the right call by not redoing _everything_